app/src/main/java/org/simple/clinic/facility/change/FacilityListItemBuilder.kt

```diff
@@ -27,7 +27,7 @@ object FacilityListItemBuilder {
     }
 
     val listItems = mutableListOf<FacilityListItem>()
-    if (nearbyFacilityItems.isNotEmpty()) {
+    if (searchQuery.isBlank() && nearbyFacilityItems.isNotEmpty()) {
```

Comment:
Is there a corresponding test for this change?

Reply:
There is, but I accidentally checked it in the previous commit. I'll fix this.